index.bs

```diff
@@ -2351,7 +2396,7 @@ It takes the following input parameters:
 : |allowCredentialDescriptorList|
 :: An optional [=list=] of {{PublicKeyCredentialDescriptor}}s describing credentials acceptable to the [=[RP]=] (possibly filtered
     by the client), if any.
-: |requireUserPresence|
+: <var ignore>requireUserPresence</var>
```

Comment:
This `<var>` is now defined and `ignore`d in two places... :)

Reply:
if I remove the ignore here, i get the annoying "only used once" warning.

Reply:
If you remove just one, yes, but not if you remove both this one and https://github.com/w3c/webauthn/pull/623#discussion_r165212044 .

Reply:
yeah, i was gonna try removing it from both next but was running out door to go sit in traffic....u r of course correct, fixed now in 
14335b1